site/content/docs/getting-started/first-app-tutorial.md

```diff
@@ -61,13 +61,13 @@ After your deployment completes your service will be up and running on AWS Farga
 Now that you've deployed your service, let's go ahead and run `copilot app delete` - this will delete all the resources Copilot set up for your application, including your ECS Service and the ECR Repository. To delete everything run:
 
 ```bash
-$ copilot app delete --env-profiles test=default
+$ copilot app delete
```

Comment:
Do you want to remove this from the `docs/` folder as well manually for now so that it goes live? normally we wouldn't do it but it'd allow us to only push this change live.

Reply:
done!